src/main/ipc.js

```diff
@@ -173,11 +173,34 @@ function init (cwd, state, logHandler) {
     e.returnValue = app.localeData
   })
 
-  ipcMain.on('updateDesktopSetting', (e, key, value) => {
+  const updateDesktopSetting = (e, key, value) => {
     const { saved } = app.state
     saved[key] = value
     app.saveState({ saved })
     render()
+  }
+  ipcMain.on('updateDesktopSetting', updateDesktopSetting)
```

Comment:
Maybe do

```js
 ipcMain.on('updateDesktopSetting', (e, k, v) => updateDesktopSetting(k, v))
```

Reply:
a new function... maybe more clearer, but us the definition is above it should be fine